src/Essentials/src/Screenshot/Screenshot.ios.cs

```diff
@@ -17,26 +17,13 @@ partial class ScreenshotImplementation : IPlatformScreenshot, IScreenshot
 		public bool IsCaptureSupported =>
 			true;
 
-		[System.Runtime.Versioning.SupportedOSPlatform("ios13.0")]
-		[System.Runtime.Versioning.SupportedOSPlatform("tvos13.0")]
```

Comment:
No longer needed because `WindowStateManager` correctly handles all the platforms.